### PR TITLE
80620 Add username on persons

### PR DESCRIPTION
**Pull request by bratli**
DevOps#80620

### CODE REVIEW

File: src/Equinor.Procosys.Library.Query/GetFunctionalRoles/PersonInFunctionalRole.cs [hunk 1]
```diff
@@ -8,6 +8,8 @@ public class PersonInFunctionalRole
 
         public string LastName { get; set; }
 
+        public string UserName { get; set; }
```
Comment (janid1967) [inline src/Equinor.Procosys.Library.Query/GetFunctionalRoles/PersonInFunctionalRole.cs:11]: why added? Not used

Comment (bratli) [reply src/Equinor.Procosys.Library.Query/GetFunctionalRoles/PersonInFunctionalRole.cs:11]: It is included in the persons under functional roles now, which we use in IPO. Am I missing something maybe?

Comment (bratli) [reply src/Equinor.Procosys.Library.Query/GetFunctionalRoles/PersonInFunctionalRole.cs:11]: New:
image
Old:
image

Comment (janid1967) [reply src/Equinor.Procosys.Library.Query/GetFunctionalRoles/PersonInFunctionalRole.cs:11]: No. Fine

**Review by janid1967 — APPROVED** [review]

**Pull request merged**